langlib/lang.string/src/main/ballerina/string.bal

```diff
@@ -54,6 +75,10 @@ public isolated function concat(string... strs) returns string = @java:Method {
 
 # Returns the code point of a character in a string.
 #
+# ```ballerina
+# "Hello, World!".getCodePoint(3) ⇒ 108
```

Comment:
Should we have both func call and method call in the examples?

Reply:
Decided to follow above approach.